common/buildcraft/transport/plug/PluggableFacade.java

```diff
@@ -47,14 +47,14 @@
     public PluggableFacade(PluggableDefinition definition, IPipeHolder holder, EnumFacing side, FacadeInstance states) {
         super(definition, holder, side);
         this.states = states;
-        isSideSolid = states.areAllStatesSolid(side);
+        isSideSolid = states.areAllStatesSolid(side) && !states.phasedStates[activeState].isHollow();
     }
 
     public PluggableFacade(PluggableDefinition def, IPipeHolder holder, EnumFacing side, NBTTagCompound nbt) {
         super(def, holder, side);
         this.states = FacadeInstance.readFromNbt(nbt, "states");
         activeState = MathUtil.clamp(nbt.getInteger("activeState"), 0, states.phasedStates.length - 1);
-        isSideSolid = states.areAllStatesSolid(side);
+        isSideSolid = states.areAllStatesSolid(side) && !states.phasedStates[activeState].isHollow();
```

Comment:
activeState can change at *any* time. Depending on it here doesn't make much sense.

Honestly neither does allowing phasing between hollow and non-hollow facades. As they aren't implemented yet this hasn't really come up, but I guess this needs fixing elsewhere as well.

Reply:
Should a condition be made for it in `FacadeInstance::areAllStatesSolid` instead?

Reply:
No, I'll fix this bit manually, as it involves changing the API.